components/dashboard/src/projects/Projects.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { Link } from "react-router-dom";
+import Header from "../components/Header";
+import projectsEmpty from '../images/projects-empty.svg';
+
+export default function() {
+    return <>
+        <Header title="Projects" subtitle="Manage recently added projects." />
+        <div>
+            <img alt="Projects (empty)" className="h-44 mt-24 mx-auto" role="presentation" src={projectsEmpty} />
+            <p className="text-center text-base text-gray-500 mt-16">Add projects to enable and manage Prebuilds.<br/><a className="underline" href="https://www.gitpod.io/docs/prebuilds/">Learn more about Prebuilds</a></p>
```

Comment:
**issue**: Reminder to use the same style with `underline-thickness-thin` and `underline-offset-small` for all information links.

For example:

https://github.com/gitpod-io/gitpod/blob/c31e2f05fe3ccc7e9e1dbae0efa9454270b4026a/components/dashboard/src/workspaces/Workspaces.tsx#L124

Reply:
Good point, thanks! Will create a `a.learn-more` class and re-use that everywhere.